pandas/tests/internals/test_internals.py

```diff
@@ -584,10 +584,6 @@ def _compare(old_mgr, new_mgr):
         new_mgr = mgr.convert()
         _compare(mgr, new_mgr)
 
-        mgr = create_mgr("a, b: object; f: i8; g: f8")
```

Comment:
is this a duplicate test? our converage of convert is probably low anyhow

Reply:
This is just the only one that doesn't pass any args, so went down the old path of using maybe_convert_objects instead of soft_convert_objects.